src/domain/LoginViewModel.js

```diff
@@ -81,4 +93,47 @@ export class LoginViewModel extends ViewModel {
             this._sessionContainer.deleteSession();
         }
     }
+
+    /**
+     * This function will track change event into the loadViewModal,
+     * and then handle it by emitChange (isBusy) in this model view
+     * @returns {Object} the load view model subscription
+     */
+    _trackLoadViewModelChange () {
+        const  _onchangeHandler = () => {
+            if (!this._loadViewModel.loading) {
+                this._loadViewModelSubscription = this.disposeTracked(this._loadViewModelSubscription);
+            }
+            this.emitChange("isBusy");
+        }
+
+        return this.track(this._loadViewModel.disposableOn("change", _onchangeHandler));
+    }
+
+    /**
+     * This function requests the login flows from _sessionContainer using SessionLoadViewModel
+     * @param {String} homeServer - the homeserver url
+     */
+    _requestLoginFlows (homeServer) {
+        this._loadViewModel  = this.track(new SessionLoadViewModel({
```

Comment:
Loading the login flows should not set the load view model. The load view model is for once you are logging in, it show "fetching your conversations from the server", etc ... It would be nice to have a spinner though, so you may want to set a `homeserverLoading` flag or something.